api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -207,6 +207,75 @@ message RedisProxy {
     repeated string commands = 4;
   }
 
+  // HotKey defines the configuration of hotkey feature.
+  // [#next-free-field: 6]
+  message HotKey {
+    enum CacheType {
+      // default.
+      // Use LFU cache. Least Frequently Used eviction mode. The lowest frequency is eliminated,
+      // and the specified number of high-frequency hotkeys is reserved.
+      LFU = 0;
+    }
+
+    // Configure Cache Type.
+    CacheType cache_type = 1 [(validate.rules).enum = {defined_only: true}];
+
+    // Configure Cache Capacity. This changes the number of hot keys that can be counted at most.
+    //
+    // default: 32
+    // min: 1
+    // max: 255
+    google.protobuf.UInt32Value cache_capacity = 2 [(validate.rules).uint32 = {lte: 255 gte: 1}];
+
+    // Configure the dispatch interval for the collection operation.
+    //
+    // default: 5s
+    // min: 0.5s
+    // max: 3600s
+    google.protobuf.Duration collect_dispatch_interval = 3 [(validate.rules).duration = {
+      lte {seconds: 3600}
+      gte {nanos: 500000000}
+    }];
+
+    // Configure the dispatch interval for the attenuation operation.
+    // To avoid full heat over a period of time, we need to attenuate it regularly.
```

Comment:
Can you define "attenuate." I'm still not completely sure what this means. I think more definition in layman's terms followed by the example would help.